guides/customization/vnc.md

```diff
@@ -72,20 +72,17 @@ environment.
 
 1. Select your **Image** and associated **Tag**.
 
-   > If you're using the sample image provided above, make sure to leave the
-   > **Run as Container-based Virtual Machine** option _unchecked_; this image
-   > doesn't currently include support for [CVMs](../../environments/cvms.md).
-
 1. Click **Create Environment**
 
-## Step 5: Create a Dev URL and access the VNC
+## Option 1: Connect via web
 
-Now that you've created your environment, you'll need to create a
-[dev URL](../../environments/devurls.md) so that you can access its services.
+If your image includes [noVNC](https://github.com/novnc/noVNC), or another
+web-based client, you can use a [dev URL](../../environments/devurls.md) to
+access it securely.
 
 1. From the **environment overview** page, click **Add URL**
 
-1. Provide the **Port** number that the VNC is running on (this information is
+1. Provide the **Port** number that noVNC is running on (this information is
```

Comment:
I'm thinking we generalize this statement rather than specifically referencing noVNC, e.g:

> Provide the **Port** number that the VNC web-client is running on

Reply:
I like this so that the doc's more evergreen, though I understand we specifically use noVNC in the image